### DISCUSSION TITLE
Anthias on x64 hardware request

### DISCUSSION THREAD

**Discussion by techminer71** [Screenly/Anthias#1785] [Ideas]
Hello,

With the shortage of Raspberry Pi's continuing and seeing that the official Screenly is using x64 for their players now, I was wondering if there may be any interest or creating any interest in an x64 version of Anthias.

With having various thin clients and/or Intel Mac Minis, this would be a good source of hardware that we could use for digital signage in our area.  I have tried getting it to work on a Thin Client by changing the architecture to x64 and had gotten some aspects to boot, but could not get past the point where it pulls docker into the equation.

I think that this would expand the pool of usable devices for Anthias and get more people using it.  If not I  would have to look at an alternate product and would prefer not to do that as we have a pretty decent amount of Screenly OSE devices already.  I mainly just wanted to get the conversation started and would like to keep it going if at all possible.

Thanks

**Comment by vpetersson** [comment]
It's something we've discussed in the past. With the changes we're making (WebKit base player, moving video to vlc), this will be a lot more attainable.

**Comment by vpetersson** [comment]
This is now supported. Anthias runs on x86 64-bit PCs. Install Debian, then run:

bash <(curl -sL https://install-anthias.srly.io)

See the installation docs for details: https://github.com/Screenly/Anthias#installation

Closing as implemented.
